Pollo/Extensions/UIColor+Shared.swift

```diff
@@ -8,44 +8,30 @@
+    @nonobjc static let polloGreen = UIColor(red: 41.0 / 255.0, green: 192.0 / 255.0, blue: 158.0 / 255.0, alpha: 1.0)
+    @nonobjc static let blueGrey = UIColor(red: 158.0 / 255.0, green: 167.0 / 255.0, blue: 179.0 / 255.0, alpha: 1.0)
+    @nonobjc static let grapefruit = UIColor(red: 231.0 / 255.0, green: 99.0 / 255.0, blue: 102.0 / 255.0, alpha: 1.0)
+    @nonobjc static let lightGrey = UIColor(white: 236.0 / 255.0, alpha: 1.0)
+    @nonobjc static let lightGreen = UIColor(red: 199.0 / 255.0, green: 239.0 / 255.0, blue: 230.0 / 255.0, alpha: 1.0)
+    @nonobjc static let darkGrey = UIColor(white: 61.0 / 255.0, alpha: 1.0)
+    @nonobjc static let mediumGray = UIColor(white: 108.0 / 255.0, alpha: 1.0)
+    @nonobjc static let offWhite = UIColor(white: 250.0 / 255.0, alpha: 1.0)
     @nonobjc static let aquaMarine = UIColor(red: 66/255, green: 219/255, blue: 197/255, alpha: 1.0)
     @nonobjc static let bluishGreen = UIColor(red: 10/255, green: 161/255, blue: 127/255, alpha: 1.0)
     @nonobjc static let charcoalGrey = UIColor(red: 48.0 / 255.0, green: 48.0 / 255.0, blue: 60.0 / 255.0, alpha: 1.0)
     @nonobjc static let clickerBlack0 = UIColor(red: 31/255, green: 44/255, blue: 56/255, alpha: 1.0)
-    @nonobjc static let clickerBlack1 = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 1.0)
-    @nonobjc static let clickerBlack2 = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 0.70)
-    @nonobjc static let clickerBlack3 = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 0.60)
     @nonobjc static let clickerBlue = UIColor(red: 73/255, green: 157/255, blue: 255/255, alpha: 1.0)
-    @nonobjc static let clickerGreen0 = UIColor(red: 0.16, green: 0.75, blue: 0.62, alpha: 1.0)
-    @nonobjc static let clickerGreen1 = UIColor(red: 41/255, green: 192/255, blue: 158/255, alpha: 0.25)
-    @nonobjc static let clickerGreen2 = UIColor(red: 6/255, green: 184/255, blue: 158/255, alpha: 0.19)
-    @nonobjc static let clickerGrey0 = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 1.0)
-    @nonobjc static let clickerGrey1 = UIColor(red: 229/255, green: 231/255, blue: 237/255, alpha: 1.0)
-    @nonobjc static let clickerGrey10 = UIColor(red: 61/255, green: 61/255, blue: 61/255, alpha: 1.0)
-    @nonobjc static let clickerGrey11 = UIColor(red: 120/255, green: 120/255, blue: 120/255, alpha: 1.0)
     @nonobjc static let clickerGrey12 = UIColor(red: 67/255, green: 67/255, blue: 67/255, alpha: 1.0)
     @nonobjc static let clickerGrey13 = UIColor(red: 169/255, green: 177/255, blue: 189/255, alpha: 1.0)
     @nonobjc static let clickerGrey14 = UIColor(red: 151/255, green: 151/255, blue: 151/255, alpha: 1.0)
-    @nonobjc static let clickerGrey2 = UIColor(red: 158/255, green: 167/255, blue: 179/255, alpha: 1.0)
     @nonobjc static let clickerGrey3 = UIColor(red: 122/255, green: 129/255, blue: 139/255, alpha: 1.0)
     @nonobjc static let clickerGrey4 = UIColor(red: 247/255, green: 249/255, blue: 250/255, alpha: 1.0)
```

Comment:
Are all of these ```clickerGreyX```colors used throughout the app? If so, should we meet with design since these colors aren't in the style guide anymore and if somewhere, for example, ```UILabel.backgroundColor = clickerGrey12``` we should replace it within the new corresponding color in the UIKit. Ex: ```._clickerGrey4 -> ._lightGrey```